sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -872,17 +874,16 @@ to update an executable graph, every node in the graph must be either a kernel
 command or a host-task.
 
 The only characteristics that can differ between two topologically identical
-graphs during an update are:
+graphs during an update are the arguments to kernel nodes. For example,
+the graph may capture different values for the USM pointers or accessors used
+in the graph. It is these kernels arguments in `graph` that constitute the
+inputs & outputs to update to.
 
-* The arguments to kernel nodes may be different between graphs. For example,
-  the graph may capture different values for the USM pointers or accessors used
-  in the graph. It is these kernels arguments in `graph` that constitute the
-  inputs & outputs to update to.
+Differences in the following characteristics between two graphs during an
+update results in undefined behaviour:
 
-* The code that is executed in host-task nodes may be different between graphs,
-  and the update will reflect the code from `graph` host-tasks nodes. However,
-  any changes to executable code between kernel nodes are not counted as
-  topologically identical and is an exception with error code `invalid`.
+* Modifying the native C++ callable of a host_task node.
```

Comment:
Hmm, our usage is a little inconsistent here: we use both `host task` and `host-task`. Do we have a preference for one over the other? I can update the other usages as well.

The SYCL spec generally uses `host task` so that would probably be my preference.

Reply:
I agree with following the wording of the spec here.